GATK_CNV_Mutect2/mutect2-gatk4/mutect2-gatk4.wdl

```diff
@@ -1,4 +1,9 @@
 version 1.0
+import "https://api.firecloud.org/ga4gh/v1/tools/neovax-pipeline:LegoPlotter/versions/1/plain-WDL/descriptor" as LegoPlotter
```

Comment:
Call it from TAG public lego plotting script

Reply:
The TAG-public lego plotter is different from firecloud version. We will keep the firecloud link for now